pom.xml

```diff
@@ -228,14 +223,19 @@
                         </goals>
                     </execution>
                 </executions>
-                <configuration>
```

Comment:
Why is this removed?
We did chose port 5007 on purpose as other processes already use 5005+5006

Reply:
Fixed.